step-plans/step-plans-core/src/main/java/step/core/artefacts/handlers/ArtefactHandler.java

```diff
@@ -106,7 +106,7 @@ public void createReportSkeleton(ReportNode parentReportNode, ARTEFACT artefact,
 			artefact.setNameDynamically();
 			reportNode.setName(getReportNodeName(artefact));
 			ArtefactFilter filter = context.getExecutionParameters().getArtefactFilter();
-			if(filter!=null&&!filter.isSelected(artefact)) {
+			if((filter!=null&&!filter.isSelected(artefact)) || artefact.getSkipNode().get()) {

```

Comment:
We could have common method for this check as the same is done in the execute phase